### DISCUSSION TITLE
colorama version limit problem

### DISCUSSION THREAD

**Discussion by kimkihoon0515** [aws/aws-cli#7645] [Q&A] [answered]
Refer to https://github.com/aws/aws-cli/pull/7402
It seems colorama 0.4.6 is now allowed but according to https://github.com/aws/aws-cli/blob/develop/setup.py, but it's not changed yet?

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
Hi @kimkihoon0515, thanks for reaching out.  It looks like the Colorama version limit update was for CLI v2, and the file you linked is for CLI v1.  CLI v2 is the most recent major version, and is recommended for general use.  The migration guide can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration.html), which includes a list of breaking changes.  Please let me know if you have any other questions.
